### PR TITLE
Add Members Section

### PR DESCRIPTION
**Pull request by heytulsiprasad**
This PR is a **WIP (Work in Progress)**. I have coded the section by making a ```member.js``` react component, and used _sass_ for writing the ```main.css```. Still as of now, the code ain't running, I might need some help in linking the ```main.css``` file to the ```member.js``` component. Thanks in advance!

### CODE REVIEW

**Comment by heytulsiprasad** [comment]
Annotation 2019-12-06 012549

This is how it is supposed to look after the style sheets are linked.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1291 @@
...
+      "version": "1.1.1",
+      "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
+      "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q=="
+    },
+    "ajv": {
+      "version": "6.10.2",
+      "resolved": "https://registry.npmjs.org/ajv/-/ajv-6.10.2.tgz",
+      "integrity": "sha512-TXtUUEYHuaTEbLZWIKUr5pmBuhDLy+8KYtPYdcV8qC+pOZL+NKqYwvWSRrVXHn+ZmRRAu8vJTAznH7Oag6RVRw==",
+      "requires": {
+        "fast-deep-equal": "^2.0.1",
```
Comment (aquibbaig) [inline package-lock.json:15]: Add this to .gitignore, else do a `git checkout` for this file

Comment (aquibbaig) [reply package-lock.json:15]: Same implies for `website/package-lock.json`

File: website/pages/en/member.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    <h3 className="heading-tertiary">Lorem Ipsum</h3>
+                    <h4 className="heading-caption">Dolor Sit Amet</h4>
+                    <p className="general-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit possimus vel iste ex aliquid dignissimos sapiente eaque aperiam tempore iure dolores, excepturi, magnam adipisci enim.</p>
+                </div>
+            </div>
+        </section>
+    )
+}
+
+export default Members;
```
Comment (aquibbaig) [inline website/pages/en/member.js]: Add trailing whitespaces to the end of the file

File: website/static/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="icon" href="data:,">
+</head>
+
+<body>
+    <div id="root"></div>
+    <script src="../src/index.js" type="text/jsx"></script>
+</body>
+    
+</html>
```
Comment (aquibbaig) [inline website/static/index.html]: Add trailing whitespaces(One more line)

File: website/static/sass/abstracts/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@mixin clearfix {
+    &::after {
+        content: "";
+        clear: both;
+        display: table;
+    }
+}
```
Comment (aquibbaig) [inline website/static/sass/abstracts/_mixins.scss]: Here

File: website/static/sass/abstracts/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+$grid-width: 114rem;
+$gutter-vertical: 8rem;
+$gutter-horizontal: 6rem;
+
+// FONT:
+$default-font-size: 1.6rem;
+
+// COLORS:
+$color-primary: #1c8eff;
+$color-secondary-text: #24292e;
```
Comment (aquibbaig) [inline website/static/sass/abstracts/_variables.scss]: Here

File: website/static/sass/base/_typography.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+.heading-caption {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 13px;
+    line-height: 16px;
+    color: $color-secondary-text;
+    display: block;
+    font-weight: 600;
+    text-align: center;
+
+}
```
Comment (aquibbaig) [inline website/static/sass/base/_typography.scss]: Trailing line

File: website/static/sass/base/_utilities.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    margin-bottom: 4rem !important;
+}
+
+.u-margin-top-big {
+    margin-top: 8rem !important;
+}
+
+.u-margin-top-huge {
+    margin-top: 10rem !important;
+}
```
Comment (aquibbaig) [inline website/static/sass/base/_utilities.scss]: Here

File: website/static/sass/pages/_home.scss [hunk 8]
```diff
@@ -0,0 +1,4 @@
+.section-members {
+    position: relative;
+    padding: 5rem;
+}
```
Comment (aquibbaig) [inline website/static/sass/pages/_home.scss]: And this too

File: yarn.lock [hunk 9]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (aquibbaig) [inline yarn.lock:4]: `git checkout` this file too

**Review by aquibbaig — CHANGES_REQUESTED** [review]
Please update the PR w.r.t. changes requested

**Comment by heytulsiprasad** [comment]
## Work Did
* Added whitespace after files, in commit [2d10abi](https://github.com/zairza-cetb/bench-routes-web/pull/10/commits/2d10ab1318cf312d199aa483601da63ba49ad116).
* Fetched the upstream from repo

## Concerns
*  `npm run start` failing in local master, with the following error:
```
Hp@LAPTOP-4CF3PCHH MINGW64 ~/Desktop/work/bench-routes-web/website (master)
$ npm run start

> @ start C:\Users\Hp\Desktop\work\bench-routes-web\website
> docusaurus-start

LiveReload server started on port 35729
Failed to start Docusaurus server: SyntaxError: 'import' and 'export' may appear only with 'sourceType: "module"' (16:0)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ start: `docusaurus-start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Hp\AppData\Roaming\npm-cache\_logs\2019-12-18T14_52_21_336Z-debug.log
```

**Comment by heytulsiprasad** [comment]
Will be updated in a new PR, Closing this due to local conflicts.

**Pull request closed without merging**